lib/linguist/languages.yml

```diff
@@ -1417,6 +1407,16 @@ Gettext Catalog:
   tm_scope: source.po
   ace_mode: text
   language_id: 129
+Gherkin:
+  type: programming
+  extensions:
+  - ".feature"
+  tm_scope: text.gherkin.feature
+  aliases:
+  - cucumber
```

Comment:
Yes.  Bison is just the GNU project implementation of the Yacc language.  But I see now that there already is a `Yacc` in linguist.  Maybe it was added later.  And that Bison is grouped under Yacc.  So I don't think there's anything to do.